.github/workflows/main.yml

```diff
@@ -6,14 +6,32 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go_version: ["1.20", "1.19"]
+        go_version:
+          - "1.19"
+          - "1.20"
     steps:
       - uses: actions/checkout@v3
       - name: Setup Go
-        uses: actions/setup-go@v3.2.0
```

Comment:
Do not change here. That is #1408. I prefer that we keep those changes in separate commits. Especially as in CI results for #1408 we are seeing issues.

Reply:
Reverted